### PR TITLE
Issue 78 docs

### PR DESCRIPTION
**Pull request by mikekeke**
Better documentation.
README reworked, new tutorials added.

### CODE REVIEW

**Comment by zmrocze** [comment]
Back when I was working out how to create the `cluster-data` config files for `vasil-local-cluster` I wrote a small doc (`cluster-data/regenerate-configs.md`)
So maybe it makes sense to keep it? and now is a good moment to add it. But feel free to revert last commit where I added it.

A detail: that doc mentioned 'print-byron-mainnet-addresses.zsh' helper script that is also included in the `vasil-local-cluster` branches. But I thought its too unpolished and removed the mention, little likely that anyone would use it (?).

**Comment by zmrocze** [comment]
Good docs!

**Comment by mikekeke** [comment]
@zmrocze 
Great addition IMO.
I moved readme to `docs` to keep `cluster-data` closer to content of corresponding `cardano-wallet` dir and added separate section for network configuring in main readme.

**Review by zmrocze — APPROVED** [review]

File: docs/tasty-integration.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+Each `predicate` will be rendered as separate test case in output log.
+
+### Asserting `wallet` final `Value`
+
+To assert the final `Value` which `wallet` will have after contract execution special syntax for `wallet` initialization is required. Currently it is done by group of "init-and-assert" functions from `Test.Plutip.Contract` module. E.g.:
+
+* `initAdaAssertValue [100] 133` - initialize `wallet` with single UTxO with 100 Ada and check that after contract execution final `Value` of all `wallet`'s UTxOs is equal to 133 Ada.
+* `initAndAssertLovelaceWith [1_000_000] VGt 2_000_000` - initialize `wallet` with single UTxO with 1000000 Lovelace and check that after contract execution final `Value` of all `wallet`'s UTxOs is *greater than* 2000000 Lovelace.
+
+***One important note*** is that Plutip handles creation of collateral UTxO under the hood. So when using assertions for `Value` it is advised to wrap `wallet`'s initialization with `withCollateral` function, like this:
```
[the remark this answers is not in the document]
Comment (mikekeke) [reply docs/tasty-integration.md]: Fixed in #[7b86507](https://github.com/mlabs-haskell/plutip/pull/133/commits/7b865072ebaddc31f93127359efbe505819fe65b)

**Pull request merged**